spec/dummy_app/app/models/thing.rb

```diff
@@ -1,7 +1,7 @@
 # frozen_string_literal: true
 
 class Thing < ActiveRecord::Base
-  has_paper_trail
+  has_paper_trail extend: VersionsExtensions
```

Comment:
Interesting. I've never used `extend` before.

Reply:
Oh man, you're missing out. I use `extend` all the time to add custom methods to specific association collections. :)

The other `has_many` option — particularly `dependent` and `autosave`, could also be useful to have configurable for the `versions` association, of course — so I figured we may as well make *all* the options (the non-conflicting/not-already-possible options anyway) available to be passed in and passed on... but it was *mostly* for the `extend` that I wanted this...